packages/react-client/package.json

```diff
@@ -13,8 +13,14 @@
     "LICENSE",
     "README.md",
     "flight.js",
+    "flight-hooks.js",
     "cjs/"
   ],
+  "exports": {
+    ".": "./flight.js",
+    "./flight": "./flight.js",
+    "./flight-hooks": "./flight-hooks.js"
```

Comment:
This package is really just exposed as a way to build custom renderers but we don't even publish it.

You don't have to include an exports but if you do you need to include `./src/*` to export the source to the compiler. https://github.com/facebook/react/blob/main/packages%2Freact%2Fpackage.json#L30

It gets stripped out on publish.